features/dd-sdk-android-trace/src/main/kotlin/com/datadog/android/trace/internal/handlers/AndroidSpanLogsHandler.kt

```diff
@@ -65,7 +65,7 @@ internal class AndroidSpanLogsHandler(
         timestampMicroseconds: Long? = null
     ) {
         val logsFeature = sdkCore.getFeature(Feature.LOGS_FEATURE_NAME)
-        if (logsFeature != null) {
+        if (logsFeature != null && fields.isNotEmpty()) {
```

Comment:
Why this change is needed?

First, with this change we can have a false positive warning in LogCat about missing Logs features, when it is actually registered.

Second, we can have then a possibility to not send anything.

Reply:
Essentially this is to avoid sending a useless log: if the `fields` map is empty, then we will send a log with a default message (that just says "span log") with no custom attributes, providing no information whatsoever.
But indeed we might get a false positive logcat message, I'll fix that.